docs/readme.md

```diff
@@ -3,14 +3,14 @@
 The [windows](https://crates.io/crates/windows) and [windows-sys](https://crates.io/crates/windows-sys) crates let you call any Windows API past, present, and future using code generated on the fly directly from the [metadata describing the API](https://github.com/microsoft/windows-rs/tree/master/crates/libs/bindgen/default) and right into your Rust package where you can call them as if they were just another Rust module. The Rust language projection follows in the tradition established by [C++/WinRT](https://github.com/microsoft/cppwinrt) of building language projections for Windows using standard languages and compilers, providing a natural and idiomatic way for Rust developers to call Windows APIs.
 
 * [Getting started](https://kennykerr.ca/rust-getting-started/)
-* [Samples](https://github.com/microsoft/windows-rs/tree/0.48.0/crates/samples)
+* [Samples](https://github.com/microsoft/windows-rs/tree/0.51.1/crates/samples)
```

Comment:
@ChrisDenton I think it's fine since this link here will point to the "current" release on crates.io for each respective version and crate. The samples readme is mainly trying to describe how to get to the right link for any particular version.

Reply:
Agreed. I think that's fine then.